loader/loader_test.go

```diff
@@ -2173,6 +2190,40 @@ services:
 	assert.ErrorContains(t, err, `invalid value "some_string", the only value allowed is 'all' or a number`)
 }
 
+func TestServiceDeviceRequestCountAndDeviceIdsExclusive(t *testing.T) {
+	_, err := loadYAML(`
+name: service-device-request-count-type
+services:
+  hello-world:
+    image: redis:alpine
+    deploy:
+      resources:
+        reservations:
+          devices:
+            - driver: nvidia
+              capabilities: [gpu]
+              count: 2
+              device_ids: ["my-device-id"]
+`)
+	assert.ErrorContains(t, err, `invalid "count" and "device_ids" are attributes are exclusive`)
```

Comment:
Question: why can't `capabilities` and `device_ids` be defined at the same time?